modin/error_message.py

```diff
@@ -33,17 +33,16 @@ def not_implemented(cls, message=""):
         )
 
     @classmethod
-    @logger_decorator("MODIN-ERROR", "ErrorMessage.single_warning", "debug")
     def single_warning(cls, message):
         message_hash = hash(message)
+        get_logger().debug(f"Modin Warning: Single Warning: {message}")
```

Comment:
Single warning prevents a warning from being displayed multiple times, but here we'll be displaying the fact that a warning happened even if the warning isn't printed - is that ok, or do you think people would find it confusing that the logger is saying a warning has been raised but there's no corresponding warning?